content/blog/tidymodels-survival-analysis/index.Rmd

```diff
@@ -0,0 +1,129 @@
+- Area under the ROC curve on the survival probabilities via `auc_roc_survival()`
+
+The probability of survival is always defined _at a certain point in time_. We call that time point the evaluation time because it is then also the time point at which we want to evaluate model performance. Metrics that work on the survival probabilities are also called _dynamic metrics_ and you read more about them here:
+
+- [Dynamic Performance Metrics for Event Time Data](https://www.tidymodels.org/learn/statistics/survival-metrics/)
+- [Accounting for Censoring in Performance Metrics for Event Time Data](https://www.tidymodels.org/learn/statistics/survival-metrics-details/)
+
+The evaluation time is also the best example to illustrate the changes necessary to the framework. Most of them were under the hood but the evaluation time is user-facing. Let's take a look at that.
+
+While the need for evaluation times is dependent on type of metric, it is not actually specified as an argument to the metric functions. Like yardstick's other metrics, those take pre-made predictions as the input. So where do you specify it then?
+
+You need to specify it to directly predict survival probabilities, via `predict()` or `augment()`. We introduced the corresponding `eval_time` argument first for fitted models in [parsnip and censored](https://www.tidyverse.org/blog/2023/04/censored-0-2-0/#introducing-eval_time) and have added it now for workflows. 
+
+You also need to specify it for the tuninging functions `tune_*()` from tune and finetune as they will predict survival probabilities as part of the tuning process. 
+
+And lastly, the `eval_time` argument now shows up when working with tuning/resampling results such as in `show_best()` or `autoplot()`. Those changes span the packages generating and working with resampling results: tune, finetune, and workflowsets.
+
+As we said, plenty of changes under the hood but you shouldn't need to notice them. Everything else should work "as usual", allowing the same ease and flexibility in combining tidymodels functionality for censored regression as for classification and regression.
+
+## The pieces come together: A case study
+
+To see it all in action, check out the case study [How long until building complaints are dispositioned?](https://www.tidymodels.org/learn/statistics/survival-case-study/)! 
+
+The city of New York publishes data on complaints received by the Department of Buildings that include how long it takes for a complaint to be dealt with ("dispositioned") as well as several characteristics of the complaint. The case study covers a full analysis. We start with splitting the data into test and training set, explore different preprocessing strategies and model types via tuning, and predict with a final model. It should give you a good first impression of how to use tidymodels for predictive survival analysis.
```

Comment:
```suggestion
The city of New York publishes data on complaints received by the Department of Buildings that include how long it takes for a complaint to be dealt with ("dispositioned") as well as several characteristics of the complaint. The case study covers a full analysis. We start with splitting the data into test and training sets, explore different preprocessing strategies and model types via tuning, and predict with a final model. It should give you a good first impression of how to use tidymodels for predictive survival analysis.
```

Reply:
I might propose merging this paragraph and the sentence above it?

Reply:
I wanted that link to be easier to spot when scanning the text quickly. I feel like it disappears too much if it's surrounded by other text on both sides, hence this split into two paragraphs.

Reply:
Okay, you got it :)